### PR TITLE
Update Kiali guide for v0.7+ and Istio 1.0

### PR DESCRIPTION
**Pull request by cfcosta**
This includes, but is not limited to:

* New images for all sections with the new layout
* New sections for Workloads and Applications

What is missing:

* [x] Distributed Tracing section, it's behaving weirdly on my machine.

### CODE REVIEW

**Comment by cfcosta** [comment]
@rafabene @lordofthejars can you review?

**Comment by lordofthejars** [comment]
Yes, I'll do it. Yesterday I installed Kiali as it says in site kiali.io
and it worked but I'll do it following the tutorial instructions.

File: documentation/modules/advanced/pages/kiali.adoc [hunk 1]
```diff
@@ -32,21 +36,29 @@ Install Kiali with the following commands:
...
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
-   IMAGE_NAME=kiali/kiali \
-   IMAGE_VERSION=latest \
-   NAMESPACE=istio-system \
-   VERSION_LABEL=master \
-   VERBOSE_MODE=4 envsubst | oc create -n istio-system -f -
+# URLS for Jaeger and Grafana
+JAEGER_URL="https://jaeger-query-istio-system.$(minishift ip).nip.io"
+GRAFANA_URL="https://grafana-istio-system.$(minishift ip).nip.io"
+VERSION_LABEL="v0.7"
```
Comment (lordofthejars) [inline documentation/modules/advanced/pages/kiali.adoc]: update to 0.7.2

File: documentation/modules/advanced/pages/kiali.adoc [hunk 2]
```diff
@@ -32,21 +36,29 @@ Install Kiali with the following commands:
...
+
+# Installs Kiali's configmap
+curl https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/kiali-configmap.yaml | \
+  VERSION_LABEL=${VERSION_LABEL} \
+  JAEGER_URL=${JAEGER_URL}  \
+  GRAFANA_URL=${GRAFANA_URL} envsubst | oc create -n istio-system -f -
```
Comment (lordofthejars) [inline documentation/modules/advanced/pages/kiali.adoc]: Can you add at requirements page that `envsubst` is required and how to install it?

**Comment by cfcosta** [comment]
Done for both changes, @lordofthejars .

**Pull request merged**